open_feature/provider/no_op_provider.py

```diff
@@ -19,7 +19,7 @@ def get_boolean_details(
         evaluation_context: EvaluationContext = None,
     ):
         return FlagEvaluationDetails(
-            key=key,
+            flag_key=key,
```

Comment:
Should the argument to this function change to `flag_key` too?

Reply:
Not according to the spec or the other sdk's https://github.com/open-feature/java-sdk/blob/main/src/main/java/dev/openfeature/sdk/FeatureProvider.java and https://github.com/open-feature/go-sdk/blob/main/pkg/openfeature/provider.go

Reply:
Ignore the above, agreed that the spec suggests using flag_key here https://github.com/open-feature/spec/blob/main/specification/sections/01-flag-evaluation.md#requirement-131 this has now been updated